AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1515,40 +1515,36 @@ - (void)_removeSubnode:(ASDisplayNode *)subnode
   [subnode __setSupernode:nil];
 }
 
+// NOTE: You must not called this method while holding the receiver's property lock. This may cause deadlocks.
```

Comment:
Yeah, this is needed pretty soon. Seems like a bit of technical debt for locking procedures.